README.md

```diff
@@ -55,34 +55,39 @@ You can run Docker commands to manage [images](https://docs.docker.com/engine/re
 
 ### Docker Compose
 
-[Docker Compose](https://docs.docker.com/compose/) lets you define and run multi-container applications with Docker. Visual Studio Code's experience for authoring `docker-compose.yml` is very rich, providing IntelliSense for valid Docker compose directives:
+[Docker Compose](https://docs.docker.com/compose/) lets you define and run multi-container applications with Docker. Our [Compose Language Service](https://github.com/microsoft/compose-language-service) in the Docker extension gives you IntelliSense and tab completions when authoring `docker-compose.yml` files.
 
- ![Docker Compose IntelliSense](resources/readme/dockercomposeintellisense.png)
+ ![Docker Compose IntelliSense](resources/readme/tab-completions.gif)
 
-For the `image` directive, you can press <kbd>ctrl+space</kbd> and VS Code will query the Docker Hub index for public images:
 
- ![Docker Compose image suggestions](resources/readme/dockercomposeimageintellisense.png)
+We also provide tooltips when you hover over a Docker Compose YAML attribute.
 
-VS Code will first show a list of popular images along with metadata such as the number of stars and description. If you continue typing, VS Code will query the Docker Hub index for matching images, including searching public profiles. For example, searching for 'Microsoft' will show you all the public Microsoft images.
+ ![Docker Compose Tooltips](resources/readme/hover-support.jpg)
```

Comment:
It's subtle but the highlighting in the hover image is a little confusing. When you took the screenshot, `./Dockerfile` was getting highlighted (maybe by an unfinished completion or the word finding things that VSCode has?)

It makes it harder to notice that `image` is also highlighted, by the hover help.

![image](https://user-images.githubusercontent.com/36966225/141482924-3aa974b3-691c-4b5d-833d-7e7bf6c59996.png)

I guess I'd retake this screenshot. It may also be a good idea to edit in a mouse cursor to make it a little clearer where the pointer is.

Reply:
Okay perfect I'll get on that!